### ISSUE TITLE
Modded recipes appear in JEI but are not craftable

### ISSUE THREAD

**Issue by Magicman432** [EnigmaticaModpacks/Enigmatica2ExpertSkyblock#600]
### Modpack Version

1.37

### Describe your issue.

This is the first time I am filing a bug report, so apologies for any errors, but I recently began to play E2ES and found that while JEI displays both the vanilla and the modifed/tweaked recipes, only the vanilla recipes are craftable. For example, the bucket recipe using plates shows in JEI but is uncraftable, but the recipe with normal iron ingots works (I have attached pictures of this from a newly installed singleplayer instance). I have directly confirmed in the same instance that the recipes for the enderio basic chassis, and the ex nihilo sieve, both also have the same issue, but obviously I have not confirmed for all tweaked recipes. Thanks for any help!  
![image](https://user-images.githubusercontent.com/59271329/147839464-e1d353a5-16fd-4b83-8173-f768919791b9.png)
![image](https://user-images.githubusercontent.com/59271329/147839466-ea3f0e5f-a943-4b2b-aa77-11cf59bdf0ba.png)
![image](https://user-images.githubusercontent.com/59271329/147839468-0fa65eaa-f54b-41a9-b692-06430fa4f26f.png)

### Latest Log

https://gist.github.com/Magicman432/67f5eedea55ffce4fdc0473cd21d186c

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by NillerMedDild** [comment]
Thank you for the report :)
Possibly silly question, but are you certain this happened in singleplayer? This bug is usually caused by the server lacking the scripts folder.

**Comment by Magicman432** [comment]
Hmm, that is odd, but yes I am very certain I am in single player, I haven't even opened to LAN or anything like that. I will say that in all of the instances so far where I have observed this, there is a scripts folder in the pack folder, and those scripts appear to be accurate, as I checked the ex nihilo one and saw the tweaked recipe with the mana infused gears, and such. 

However, as I am writing this I do have one new piece of news. As I am sure happens with a lot of these bugs, I have definitely found it to be on my end, and I probably should have included it before, my apologies, but it seems to be an issue with the launcher I used. I use GDLauncher, and while that has never been an issue before, the problem seems to have been solved after using overwolf. Again, I am quite sorry about not including that previously. I am still quite confused on what GDLauncher is doing, especially since this issue persisted even after importing the zip from the curseforge website, but I don't think you would be the person to help me with that.  

Thanks for your help regardless!
